frontend/src/pages/notebookController/SpawnerPage.tsx

```diff
@@ -340,86 +367,100 @@ const SpawnerPage: React.FC<SpawnerPageProps> = React.memo(({ setStartModalShown
   };
 
   return (
-    <>
-      <Form className="odh-notebook-controller__page odh-notebook-controller__page-form">
-        <FormSection title="Notebook image">
-          <FormGroup fieldId="modal-notebook-image">
-            <Grid sm={12} md={12} lg={12} xl={6} xl2={6} hasGutter>
-              {images.sort(checkOrder).map((image) => (
-                <GridItem key={image.name}>
-                  <ImageSelector
-                    image={image}
-                    selectedImage={selectedImageTag.image}
-                    selectedTag={selectedImageTag.tag}
-                    handleSelection={handleImageTagSelection}
-                  />
-                </GridItem>
-              ))}
-            </Grid>
-          </FormGroup>
-        </FormSection>
-        <FormSection title="Deployment size">
-          {sizeOptions && (
-            <FormGroup label="Container size" fieldId="modal-notebook-container-size">
-              <Select
-                isOpen={sizeDropdownOpen}
-                onToggle={() => setSizeDropdownOpen(!sizeDropdownOpen)}
-                aria-labelledby="container-size"
-                selections={selectedSize}
-                onSelect={handleSizeSelection}
-                menuAppendTo="parent"
-              >
-                {sizeOptions}
-              </Select>
-            </FormGroup>
-          )}
-          {gpuOptions && (
-            <FormGroup label="Number of GPUs" fieldId="modal-notebook-gpu-number">
-              <Select
-                isOpen={gpuDropdownOpen}
-                onToggle={() => setGpuDropdownOpen(!gpuDropdownOpen)}
-                aria-labelledby="gpu-numbers"
-                selections={selectedGpu}
-                onSelect={handleGpuSelection}
-                menuAppendTo="parent"
-              >
-                {gpuOptions}
-              </Select>
+    <QuickStarts>
```

Comment:
Move `QuickStarts` to the tippy top of NotebookController stack.

Reply:
Updated.